src/traits.rs

```diff
@@ -671,7 +701,7 @@ pub trait PrimInt
     /// ```
     /// use num::traits::PrimInt;
     ///
-    /// let n = 0b01001100u8;
+    /// let n = 0b0100_1100u8;
```

Comment:
There is no need IMHO to introduce separator in quartets of bits. Octets are readable enough for me.